site/content/api/Device-Registration-API.md

```diff
@@ -137,16 +137,20 @@ A response to an *assertion* request contains the [Standard Response Properties]
 
 The body of the response message consists of a single *AMQP Value* section containing a UTF-8 encoded string representation of a single JSON object having the following properties:
 
-| Name             | Mandatory | Type       | Description |
-| :--------------- | :-------: | :--------- | :---------- |
-| *device-id*      | *yes*     | *string*   | The ID of the device that is subject of the assertion. |
-| *assertion*      | *yes*     | *string*   | A [JSON Web Token](https://jwt.io/introduction/) which MUST contain the device id (`sub` claim), the tenant id (private `ten` claim) and an expiration time (`exp` claim). The token MAY contain additional claims as well. A client SHOULD silently ignore claims it does not understand. |
+| Name             | Mandatory | Type          | Description |
+| :--------------- | :-------: | :------------ | :---------- |
+| *device-id*      | *yes*     | *string*      | The ID of the device that is subject of the assertion. |
+| *assertion*      | *yes*     | *string*      | A [JSON Web Token](https://jwt.io/introduction/) which MUST contain the device id (`sub` claim), the tenant id (private `ten` claim) and an expiration time (`exp` claim). The token MAY contain additional claims as well. A client SHOULD silently ignore claims it does not understand. |
+| *defaults*       | *no*      | *JSON object* | Default values to be used by protocol adapters for augmenting messages from devices with missing information like a *content type*. It is up to the discretion of a protocol adapter if and how to use the given default values when processing messages published by the device. |
```

Comment:
Why shouldn't we make it mandatory for protocol adapters to use the "defaults"?
If defaults are not desired, they can just be not set in the device registry, but if we have some protocol adapters in the future that consider them, and some not, what do we gain?

Reply:
We can always make our protocol adapters use the default properties but I do not think that we can enforce this on custom adapters.

Reply:
OK, I understand.
Additionally I haven't seen that defaults can be switched off (which makes sense to me), so I guess it should stay like it is...